### PR TITLE
[FrameworkBundle] tweak the factory class description for inlined factory services

### PR DESCRIPTION
**Pull request by xabbuh**
| Q             | A
| ------------- | ---
| Branch?       | 4.4
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Tickets       | Fix https://github.com/symfony/symfony/pull/47351#discussion_r958677734
| License       | MIT
| Doc PR        |

### CODE REVIEW

File: src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/JsonDescriptor.php [hunk 1]
```diff
@@ -219,7 +219,7 @@ private function getContainerDefinitionData(Definition $definition, bool $omitTa
                 if ($factory[0] instanceof Reference) {
                     $data['factory_service'] = (string) $factory[0];
                 } elseif ($factory[0] instanceof Definition) {
-                    $data['factory_class'] = $factory[0]->getClass() ?? 'not configured';
+                    $data['factory_class'] = sprintf('inline factory service (%s)', $factory[0]->getClass() ?? 'class not configured');
```
Comment (stof) [inline src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/JsonDescriptor.php]: should be `factory_service` being set here, not `factory_class`

File: src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/MarkdownDescriptor.php [hunk 2]
```diff
@@ -196,7 +196,7 @@ protected function describeContainerDefinition(Definition $definition, array $op
                 if ($factory[0] instanceof Reference) {
                     $output .= "\n".'- Factory Service: `'.$factory[0].'`';
                 } elseif ($factory[0] instanceof Definition) {
-                    $output .= "\n".'- Factory Class: `'.($factory[0]->getClass() ?? 'not configured').'`';
+                    $output .= "\n".sprintf('- Factory Class: inline factory service (%s)', $factory[0]->getClass() ? sprintf('`%s`', $factory[0]->getClass()) : 'not configured');
```
Comment (stof) [inline src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/MarkdownDescriptor.php]: should be `Factory Service`

**Review by stof — CHANGES_REQUESTED** [review]

File: src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/JsonDescriptor.php (hunk 1 above)
Comment (xabbuh) [reply src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/JsonDescriptor.php]: updated

File: src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/MarkdownDescriptor.php (hunk 2 above)
Comment (xabbuh) [reply src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/MarkdownDescriptor.php]: same here

**Review by stof — APPROVED** [review]

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @xabbuh.

**Pull request merged**